### DISCUSSION TITLE
Rewind Track

### DISCUSSION THREAD

**Discussion by turetafr** [AndreyPavlenko/Fermata#485] [Ideas]
In most media players throughout history, pressing the previous track button would rewind the current track, if not already within the first few seconds of the track. Additional presses of the previous track button would then, in fact, play the previous track.  I am only asking for this because I am unable to use my steering wheel control to rewind a track as I've always been used to.

**Comment by AndreyPavlenko** [comment]
You can configure the key bindings in the app settings. By default, long press acts as ff/rw, but some steering wheel control devices do not support the long press.
